pool/app/system_event/system_event.mli

```diff
@@ -22,20 +23,21 @@ end
 type t =
   { id : Id.t
   ; job : Job.t
+  ; worker_only : bool
   ; created_at : Pool_common.CreatedAt.t
   ; updated_at : Pool_common.UpdatedAt.t
   }
 
 val equal : t -> t -> Ppx_deriving_runtime.bool
 val pp : Ppx_deriving_runtime.Format.formatter -> t -> Ppx_deriving_runtime.unit
 val show : t -> Ppx_deriving_runtime.string
-val create : ?id:Id.t -> Job.t -> t
+val create : ?id:Id.t -> ?worker_only:bool -> Job.t -> t
```

Comment:
Once the argument is called "worker" and once "worker_only", I'd suggest using only one of them or creating e.g. a polymorphic variant identifier (instead of string) with 

```ocaml
type identifier =
  | `Server
  | `Worker
```